ts/src/core/ent.ts

```diff
@@ -1284,7 +1284,7 @@ export function getCursor(opts: cursorOptions) {
   }
   const convert = (d: any) => {
     if (d instanceof Date) {
-      return d.getTime();
+      return d.toISOString();
```

Comment:
this makes sense. i don't recall if this is in the cursor somewhere. does this break something?

Reply:
I don't think this will break anything unless people were storing cursors for some reason, but these should just be used with each request so it would just generate a different cursor than before and be fine.

Reply:
So, this actually did break things so I needed to change the logic. I think the reason you had done `d.getTime()` to begin with was because you were storing the cursor as `<key>:<value>`, and `d.getISOString()` returns a string with `:`s. So, when you did `cursor.split(':')` it failed. I changed this to be an array of key/value tuples that I just `JSON.stringify`ed and that works for everything.